app/controllers/items_controller.rb

```diff
@@ -2,4 +2,26 @@ class ItemsController < ApplicationController
   def index
     @items = Item.all
   end
+
+  def new
+    @item = Item.new
+  end
+
+  def create
+    @item = Item.new(item_params)
+    if @item.save
+      flash[:success] = t("items.item_added")
+      redirect_to items_path
+    else
+      flash[:danger] = t("errors.fix_errors") 
+      render 'items/new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.